noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -0,0 +1,206 @@
+// Converts the input bytes into an array of fields. A Field is ~254 bits meaning that each field can store 31 bytes.
+// This implies that M = ceil(N / 31).
+pub fn bytes_to_fields<let N: u32, let M: u32>(input: [u8; N]) -> [Field; M] {
+    assert(N <= 31 * M, "Bytes do not fit into fields");
+    let mut dst = [0; M];
+
+    for field_index in 0..M {
```

Comment:
```suggestion
    for dst_index in 0..M {
```

To me 'field index' sounds like it might be an index inside a field. I'd consider going with `src_index` and `dst_index`, which also reduces the cognitive overhead of going 'i'm converting bytes to fields, to byte index is the index of the source'.

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/8590/commits/6899a4803fb640b52e895ed02946dd68d6d8f6ad